### PR TITLE
chore(dev): add redux-devtools-extension

### PR DESCRIPTION
**Pull request by lex111**
For better Redux debugging and for React Native Debugger application work https://github.com/jhen0409/react-native-debugger

### CODE REVIEW

**Comment by machour** [comment]
> I was blind, but now I see.

Really nice! 👍 👍 👍

**Comment by housseindjirdeh** [comment]
Definitely a fan of the dev-tools, have used it quite a bit in the past few weeks for React web.

Just a few concerns. We still have logger _and_ reactotron in the app. Are we including too many logging middleware and can potentially limit it to 1? Also will this boot up the dev tools every single time we run the simulator?

**Comment by lex111** [comment]
@housseindjirdeh actually, no, I think. Reactotron is different, it differs from React Native Debugger, and both of these applications do not work together, so `composeWithDevTools` is not added when Reactotron is enabled.
I can, like the logger and Reactotron, add an environment variable, such as `devtools`, i.e. `start: android: devtools`, then `redux-devtools-extension` will be run, so do?

**Comment by lex111** [comment]
@housseindjirdeh what do we do, can we merge already?

File: package.json [hunk 1]
```diff
@@ -76,7 +76,8 @@
     "redux-persist": "^4.3.1",
     "redux-persist-transform-encrypt": "^1.0.2",
     "redux-thunk": "^2.2.0",
-    "reselect": "^3.0.1"
+    "reselect": "^3.0.1",
+    "redux-devtools-extension": "^2.13.2"
```
Comment (housseindjirdeh) [inline package.json]: Can we have this as a `devDependency`?

Comment (lex111) [reply package.json]: Hmm, actually I looked at other repositories, and there the dependency `redux-devtools-extension` is in just `dependencies`. Given that this only works when the debugging mode is enabled, then I understand that there is no point in adding a script command, I may be mistaken, but as I understand it, this is so.

Comment (housseindjirdeh) [reply package.json]: Perfect no worries, if you don't think it makes sense to add a script then that's perfectly fine :)

Yeah and I've noticed the same with `redux-logger` and I'm still no sure why those two usually live as dependencies :thinking: Let's circle back to that after some time.

**Pull request merged**